### PR TITLE
WRP-17865: Update package dependencies to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] I have run automated CLI testing and it is passed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated dependencies to the latest minor version.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Dependencies now use the latest minor version

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
There were breaking changes affecting this repo. https://babeljs.io/blog/2023/05/26/7.22.0#renamed-packages  https://github.com/babel/babel/releases
Some babel plugins needed to be renamed.

### Links
[//]: # (Related issues, references)
WRP-17865

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

**Comment by daniel-stoian-lgp** [comment]
please list the breaking changes from eslint and add a changelog

**Review by daniel-stoian-lgp — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## [unreleased]
+
+* Renamed several used babel plugins from `@babel/plugin-proposal` to `@babel/plugin-transform` due to their renaming in `@babel` version `7.22.0`.
```
Comment (hong6316) [inline CHANGELOG.md:3]:
```suggestion
* Updated dependencies and renamed several used babel plugins from `@babel/plugin-proposal` to `@babel/plugin-transform`.
```

**Comment by hong6316** [comment]
Nice jobs :)
Are there any missing changes in our repo among the links below?
https://babeljs.io/blog/2023/05/26/7.22.0#renamed-packages 
https://github.com/babel/babel/releases

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**